frontend/src/components/TagSelector/react_tags.sass

```diff
@@ -0,0 +1,145 @@
+/**
+ * <div class="react-tags">
+ *   <div class="react-tags__selected">
+ *     <button class="react-tags__selected-tag">
+ *       <span class="react-tags__selected-tag-name" />
+ *     </button>
+ *   </div>
+ *   <div class="react-tags__search">
+ *     <div class="react-tags__search-wrapper">
+ *       <input class="react-tags__search-input" />
+ *       <div />
+ *     </div>
+ *     <div class="react-tags__suggestions">
+ *       <ul>
+ *         <li class="is-active">
+ *           <mark />
+ *         </li>
+ *         <li class="is-disabled">
+ *           <mark />
+ *         </li>
+ *       </ul>
+ *     </div>
+ *   </div>
+ */
```

Comment:
Why do you need this set of comments?

Reply:
unfortunaly, this code wasn't written by me, i pulled it from none-dev branch to reusing components and havent resolve yet its purpose

Reply:
Just get rid of it.

Reply:
Done, master!)